dll/win32/shell32/wine/shell32_main.c

```diff
@@ -1381,7 +1395,22 @@ BOOL WINAPI ShellAboutW( HWND hWnd, LPCWSTR szApp, LPCWSTR szOtherStuff,
     if(!DlgTemplate)
         return FALSE;
 
+#ifdef __REACTOS__
+    /*
+     * KERNEL_VERSION_STR and KERNEL_VERSION_BUILD_STR are ANSI strings so we must convert them to Unicode
+     * (wide characters) before going to concatenate the OS version strings.
+    */
+    MultiByteToWideChar(CP_ACP, 0, KERNEL_VERSION_STR, -1, szKernelVersion, _countof(szKernelVersion));
+    MultiByteToWideChar(CP_ACP, 0, KERNEL_VERSION_BUILD_STR, -1, szKernelVersionBuild, _countof(szKernelVersionBuild));
```

Comment:
Much simpler you can use macros directly where you use them, as show here:
https://git.reactos.org/?p=reactos.git;a=blob;f=base/setup/welcome/welcome.c;hb=e00ea7daddfa21ba692d6d9a22d460d79920c6fc#l1353
https://git.reactos.org/?p=reactos.git;a=blob;f=base/shell/cmd/cmd.c;hb=e00ea7daddfa21ba692d6d9a22d460d79920c6fc#l2011
https://git.reactos.org/?p=reactos.git;a=blob;f=base/shell/cmd/ver.c;hb=e00ea7daddfa21ba692d6d9a22d460d79920c6fc#l138